data/phi1014/__cts__.xml

```diff
@@ -1,4 +1,4 @@
 <ti:textgroup xmlns:ti="http://chs.harvard.edu/xmlns/cts" xmlns:cts="http://chs.harvard.edu/xmlns/cts/ti" xmlns:atom="http://www.w3.org/2005/Atom" urn="urn:cts:latinLit:phi1014">
-          <ti:groupname xml:lang="eng">Seneca, Lucius Annaeus, 55 B.C.-ca. 39 A.D.</ti:groupname>
+          <ti:groupname xml:lang="eng">Seneca, Lucius Annaeus, ca. 55 B.C.-ca. 39 A.D.</ti:groupname>
```

Comment:
We do not use years in the textgroup names.

Reply:
I had been wondering about that and had taken it out, but then I put it back in when I realized there was no other way to distinguish between the two Senecas in the Scaife TOC without the dates. It looks like in Perseus this was dealt with Seneca and Seneca the elder.  This off course is tangential @lcerrato  to my other issue regarding two text groups for Seneca the Younger #502

Reply:
@AlisonBabeu yes, it looks like the Pliny’s are elder and younger. With Seneca, we mostly mean the younger as he is the one most students read.

Reply:
@lcerrato I can imagine, so should I go with Seneca and Seneca the elder than in the CTS descriptions?